client/sand/http.go

```diff
@@ -27,12 +29,15 @@ func (c *client) rawDialer(ctx context.Context, sandNetworkID, network, address
 }
 
 type HTTPRoundTripperOpts struct {
-	TLSConfig *tls.Config
+	TLSConfig         *tls.Config
+	DisableKeepAlives bool
 }
 
 func (c *client) NewHTTPRoundTripper(ctx context.Context, id string, opts HTTPRoundTripperOpts) http.RoundTripper {
 	return &http.Transport{
-		TLSClientConfig: opts.TLSConfig,
+		TLSClientConfig:   opts.TLSConfig,
+		DisableKeepAlives: opts.DisableKeepAlives,
+		IdleConnTimeout:   10 * time.Second,
```

Comment:
`IdleConnTimeout` is not useful with `keepalives` disabled

Reply:
Ahh my bad `Timeout` config is outside the `Transport` struct, so no need to set a timeout here

Reply:
I thought you were discussing the idle timeout in case keepalives were enabled.

I'll remove it then

Reply:
Yes sorry, I thought the `Timeout` was in the Transport struct